src/nvim/ex_getln.c

```diff
@@ -1669,7 +1672,16 @@ getcmdline (
     int indent               // indent for inside conditionals
 )
 {
-  return command_line_enter(firstc, count, indent);
+  if (ui_is_external(kUICmdline)) {
+    Array args = ARRAY_DICT_INIT;
+    ui_event("cmdline_enter", args);
```

Comment:
I think we can leave the UI to decide whether they want to implement this complexity. What's the drawbacks to enable this new event?